src/shader/shaderUtils.js

```diff
@@ -7,6 +7,14 @@ export const shaderUtils = /* glsl */`
 
 	}
 
+	float schlickFresnel( float u ) {
+
+		float m = clamp( 1.0 - u, 0.0, 1.0 );
+		float m2 = m * m;
+		return m2 * m2 * m;
+
+	}
```

Comment:
What's the difference between this function and the one above? Is this the same as calling the above function with `f0 === 1.0`?

Reply:
It's the same as calling the above function with `f0 = 0.0` or calling `schlickFresnelFromIor` with an `iorRatio` of 1. This isn't actually used and I forgot to strip it out. Such a function is not necessary if constant propagation is applied during compilation.